src/components/InputName/InputName.tsx

```diff
@@ -74,18 +75,84 @@ const InputElement = styled.input`
     font-family: 'Futura Std', Arial, Helvetica, sans-serif;
   }
 `
+const Button = styled.button`
+  position: absolute;
+  border: none;
+  background: none;
+  cursor: pointer;
+  bottom: 13px;
+  right: 17px;
+  @media (min-width: 400px) {
+    right: 38px;
+  }
+`
+
+const LabelContainer = styled.div`
+  display: flex;
+`
 
 export interface InputProps {
   label: string
   placeholder: string
   name: string
+  type: string
+  validInput: boolean
+  inputValue: string
+  onChange: React.ChangeEventHandler<HTMLInputElement> | undefined
+  errorMessage?: string
 }
 
-export const Input: React.FC<InputProps> = ({ label, placeholder, name }) => {
+export const Input: React.FC<InputProps> = ({
+  label,
+  placeholder,
+  name,
+  type,
+  validInput,
+  inputValue,
+}) => {
+  const [showPassword, setShowpassword] = useState(false)
+
   return (
     <InputContainer>
-      <label className='labelStyle'>{label}</label>
-      <InputElement placeholder={placeholder} className={name} />
+      <div style={{ position: 'relative' }}>
+        {!validInput ? (
+          <LabelContainer>
+            <label className='labelStyle'>{label}</label>
```

Comment:
This is said to be against styled components. Can we please make a little change to this line.

Reply:
Great spot!